receiver/cloudfoundryreceiver/receiver.go

```diff
@@ -25,23 +26,35 @@ const (
 )
 
 var _ receiver.Metrics = (*cloudFoundryReceiver)(nil)
+var _ receiver.Logs = (*cloudFoundryReceiver)(nil)
+
+type telemetryType int8
+
+const (
+	telemetryTypeMetrics telemetryType = iota
+	telemetryTypeLogs
+	telemetryTypeTraces
+)
 
 // newCloudFoundryReceiver implements the receiver.Metrics for Cloud Foundry protocol.
 type cloudFoundryReceiver struct {
-	settings          component.TelemetrySettings
-	cancel            context.CancelFunc
-	config            Config
-	nextConsumer      consumer.Metrics
+	settings            component.TelemetrySettings
+	cancel              context.CancelFunc
+	config              Config
+	nextMetricsConsumer consumer.Metrics
+	nextLogsConsumer    consumer.Logs
```

Comment:
What do you think of just naming these `nextMetrics` and `nextLogs` instead? I'm not sure we need the type in the variable name. (Even though it was there before)